src/app/components/wrapper-components/rules-configuration/RulesConfigDescription.ts

```diff
@@ -0,0 +1,89 @@
+import { MGPValidator, MGPValidators } from 'src/app/utils/MGPValidator';
+
+import { NamedRulesConfig, RulesConfig } from 'src/app/jscaip/RulesConfigUtil';
+import { Localized } from 'src/app/utils/LocaleUtils';
+import { MGPSet } from 'src/app/utils/MGPSet';
+import { Utils } from 'src/app/utils/utils';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { GobanConfig, defaultGobanConfig } from 'src/app/jscaip/GobanConfig';
+
+export class RulesConfigDescription<R extends RulesConfig = RulesConfig> {
+    // TODO FOR REVIEW: pour les config non type, si j'enlève le "= RulesConfig" ça devient obligé de mettre ça a plein d'endroit, moyen ?
```

Comment:
ouais moyen, mais renommer/aliaser RulesConfig en EmptyRulesConfig pour être clair ?
Une config étend la config vide, par défaut on a la config vide, ça fait sens non ?

Reply:
EmptyRulesConfig is born !